src/plugins/intel_gna/src/backend/gna_limitations.hpp

```diff
@@ -78,6 +79,20 @@ inline bool IsTransposeSupported(const std::vector<size_t>& shape) {
     return min <= 8 && max % 8 == 0 && max >= 8 && max <= transposeMaxSize;
 }
 
+inline const size_t getMemoryAlignmentBytes(target::DeviceVersion target) {
+    const std::unordered_map<target::DeviceVersion, size_t> mem_alignment_map{
```

Comment:
any call to this inline function creates new map. do we like it?

Reply:
fixed